### PR TITLE
Package mm.0.8.3

### PR DESCRIPTION
**Pull request by toots**
### `mm.0.8.3`
The mm library contains high-level to create and manipulate multimedia streams (audio, video, MIDI)

---
* Homepage: https://github.com/savonet/ocaml-mm
* Source repo: git+https://github.com/savonet/ocaml-mm.git
* Bug tracker: https://github.com/savonet/ocaml-mm/issues

---
:camel: Pull-request generated by opam-publish v2.2.0

### CODE REVIEW

File: packages/mm/mm.0.8.3/opam [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+  "The mm library contains high-level to create and manipulate multimedia streams (audio, video, MIDI)"
+maintainer: ["The Savonet Team <savonet-users@lists.sourceforge.net>"]
+authors: ["Samuel Mimram <smimram@gmail.com>"]
+license: "LGPL-2.1-or-later"
+homepage: "https://github.com/savonet/ocaml-mm"
+bug-reports: "https://github.com/savonet/ocaml-mm/issues"
+depends: [
+  "ocaml" {>= "4.08"}
+  "ocaml" {with-test & >= "4.12"}
+  "dune" {>= "3.6"}
```
Comment (mseri) [inline packages/mm/mm.0.8.3/opam:13]:
```suggestion
  "dune" {>= "3.6.1"}
```

Due to
```

#=== ERROR while compiling mm.0.8.3 ===========================================#
# context              2.2.0~alpha~dev | linux/x86_64 | ocaml-base-compiler.4.08.1 | pinned(https://github.com/savonet/ocaml-mm/archive/refs/tags/v0.8.3.tar.gz)
# path                 ~/.opam/4.08/.opam-switch/build/mm.0.8.3
# command              ~/.opam/opam-init/hooks/sandbox.sh build dune build -p mm -j 31 @install
# exit-code            1
# env-file             ~/.opam/log/mm-7-5947e9.env
# output-file          ~/.opam/log/mm-7-5947e9.out
### output ###
# (cd _build/default && /home/opam/.opam/4.08/bin/ocamlc.opt -w -40 -g -bin-annot -I src/.mm_video.objs/byte -I src/.mm_audio.objs/byte -I src/.mm_base.objs/byte -I src/.mm_image.objs/byte -no-alias-deps -open Mm_video -o src/.mm_video.objs/byte/mm_video__Video.cmi -c -intf src/video.mli)
# File "src/video.mli", line 117, characters 23-49:
# 117 | module MakeCanvas (_ : Mm_image.Image.CanvasImage) : sig
#                              ^^^^^^^^^^^^^^^^^^^^^^^^^^
# Alert deprecated: module Mm_image.Mm_image
# this module is shadowed
# File "src/video.mli", line 117, characters 23-49:
# 117 | module MakeCanvas (_ : Mm_image.Image.CanvasImage) : sig
#                              ^^^^^^^^^^^^^^^^^^^^^^^^^^
# Error: Unbound module Mm_image.Image
```

**Comment by mseri** [comment]
There is an out of memory error in the 32 bit versions tests:
```
# - scale... Fatal error: exception Out of memory
# Raised by primitive operation at Mm_image__ImageYUV420.to_BMP in file "src/imageYUV420.ml", line 159, characters 2-19
# Called from Dune__exe__Test.(fun) in file "examples/test.ml", line 334, characters 24-46
# Called from Dune__exe__Test.time in file "examples/test.ml", line 33, characters 4-8
# Called from Dune__exe__Test in file "examples/test.ml", line 329, characters 2-256
```
Should we do anything about it?

File: packages/mm/mm.0.8.3/opam [hunk 2]
```diff
@@ -0,0 +1,46 @@
+# This file is generated by dune, edit dune-project instead
+opam-version: "2.0"
+synopsis:
+  "The mm library contains high-level to create and manipulate multimedia streams (audio, video, MIDI)"
```
Comment (raphael-proust) [inline packages/mm/mm.0.8.3/opam:4]: High-level what? I think there's a word missing there.

Comment (toots) [reply packages/mm/mm.0.8.3/opam:4]:
```suggestion
  "The mm library contains high-level APIs to create and manipulate multimedia streams (audio, video, MIDI)"
```

Comment (toots) [reply packages/mm/mm.0.8.3/opam]: Good point thanks!

**Comment by toots** [comment]
> There is an out of memory error in the 32 bit versions tests:
> 
> ```
> # - scale... Fatal error: exception Out of memory
> # Raised by primitive operation at Mm_image__ImageYUV420.to_BMP in file "src/imageYUV420.ml", line 159, characters 2-19
> # Called from Dune__exe__Test.(fun) in file "examples/test.ml", line 334, characters 24-46
> # Called from Dune__exe__Test.time in file "examples/test.ml", line 33, characters 4-8
> # Called from Dune__exe__Test in file "examples/test.ml", line 329, characters 2-256
> ```
> 
> Should we do anything about it?

I would ignore it. Most of our tests are supposed 

> There is an out of memory error in the 32 bit versions tests:
> 
> ```
> # - scale... Fatal error: exception Out of memory
> # Raised by primitive operation at Mm_image__ImageYUV420.to_BMP in file "src/imageYUV420.ml", line 159, characters 2-19
> # Called from Dune__exe__Test.(fun) in file "examples/test.ml", line 334, characters 24-46
> # Called from Dune__exe__Test.time in file "examples/test.ml", line 33, characters 4-8
> # Called from Dune__exe__Test in file "examples/test.ml", line 329, characters 2-256
> ```
> 
> Should we do anything about it?

This test hasn't changed since the previous release and it was passing by then: https://opam-ci.ci3.ocamllabs.io/github/ocaml/opam-repository/commit/9ef4c69c6c6fe2888e1abe93fbbdb4acc9a25f9b

I'd consider this as a arch issue/temporary glitch?

**Comment by toots** [comment]
Actually sorry the tests were already failing during the last release: https://opam-ci.ci3.ocamllabs.io/github/ocaml/opam-repository/commit/9ef4c69c6c6fe2888e1abe93fbbdb4acc9a25f9b

**Comment by mseri** [comment]
Indeed, we discussed it on release 0.3.1 when it appeared: https://github.com/ocaml/opam-repository/pull/21420

Please upstream the changes in the opam file

**Pull request merged**